docs/src/user-guide/formatters/index.md

```diff
@@ -83,28 +97,48 @@ function addOne(i) {
 
 ### html
 
+TODO: quick overview
+
+Example output:
+
 <iframe src="html-formatter-example.html" width="100%" height="460px"></iframe>
 
 ### jslint-xml
 
+TODO: quick overview
```

Comment:
There's not much context in https://github.com/eslint/eslint/issues/15, but I'm pretty sure this is another one you'd use if you need to pass data to a system that expects a particular format.

Reply:
There is a [JSLint plugin for Jenkins](https://plugins.jenkins.io/jslint/) that expects a specific XML format to produce reports. This formatter type outputs data in the format that the Jenkins plugin expects. This was necessary so that people could swap out JSLint for ESLint in their Jenkins projects.